### ISSUE TITLE
NginxPM Control panel behind NginxPM proxy, API stop working....

### ISSUE THREAD

**Issue by alexsalex** [NginxProxyManager/nginx-proxy-manager#3058]
When I put the PM behind the proxy, API stopped working. Could you tell me if I did something wrong?

<img width="506" alt="Screenshot 2023-07-14 at 2 14 59 PM" src="https://github.com/NginxProxyManager/nginx-proxy-manager/assets/17325366/7caba6db-113e-4fa4-9fbe-9628d7d41480">

**Comment by agent-ck** [comment]
Not to bother on an older topic, but maybe you're using NPM as docker container?

In that case you should use the IP address of the docker host instead of localhost. Then it will work fine.

When you're using localhost inside a container, then NPM will try to access the target into the same container, not across involved containers. 

A trap that has often been my downfall, especially when I was tired, but tried to do a "quick" fix to a minor problem which wasn't really necessary to be fixed ...

**Comment by alexsalex** [comment]
Yes, I use docker. I do not publish port 81 in public. For NPM containet I publish ONLY 80 and 443 port. And would like to redirect host from 80 (or 443) port to internal docker network to NPM container to port 81.

Everything works fine exept API....
